src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Route, BrowserRouter as Router, Routes } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import CountryData from './components/CountryData';
+import 'bootstrap/dist/css/bootstrap.css';
+
+function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/country/:name" element={<CountryData />} />
+      </Routes>
+    </Router>
+  );
+}
```

Comment:
Great job @Kayonga99 🤝

- [ ] [Optional] Kindly use ES6 syntax for all your functions and methods across your project, as this is regarded as a good practice for Javascript.